frameioclient/download.py

```diff
@@ -1,18 +1,49 @@
-import requests
-import math
 import os
+import math
+import requests
+
+from .exceptions import DownloadException
 
 class FrameioDownloader(object):
   def __init__(self, asset, download_folder):
     self.asset = asset
     self.download_folder = download_folder
 
+  def get_download_key(self):
+    try:
+      url = self.asset['original']
+    except KeyError:
+      if self.asset['is_session_watermarked'] == True:
+        resolution_map = dict() 
+        for resolution_key, download_url in sorted(self.asset['downloads'].items()):
+          resolution = resolution_key.split("_")[1] # Grab the item at index 1 (resolution)
+          try:
+            resolution = int(resolution)
+            if download_url is not None:
+              resolution_map.update({
+                resolution: download_url
+              })
+          except ValueError:
+            continue
```

Comment:
small suggestions, non-blocking. 
I think we can `continue` early. also, we should probably only wrap the code we want to `catch` in the `try/catch` block. 

```suggestion
         if download_url is None:
           continue
          resolution = resolution_key.split("_")[1] # Grab the item at index 1 (resolution)
          try:
            resolution = int(resolution)
          except ValueError:
            continue
          
          resolution_map.update({ resolution: download_url }) # resulotion_map[resolution] = download_url ?
```